docs/src/constructors.md

```diff
@@ -67,6 +68,18 @@ fm1reml = fit(MixedModel, fm, dyestuff, REML=true)
     @test VarCorr(fm1reml).σρ.batch.σ[1] ≈ 42.000602
 end
 ```
+### Float-point type in the model
+
+The type of `fm1`
+```@example Main
+typeof(fm1)
+```
+includes the floating point type used internally for the various matrices, vectors, etc. that represent the model.
+At present, this will always be `Float64` because the parameter estimates are optimized using the [`NLopt` package](https://github.com/JuliaOpt/NLopt.jl) which calls compiled C code that only allows for optimization with respect to a `Float64` parameter vector.
+
+So in theory other floating point types, such as `BigFloat` or `Float32`, can be used to define a model but in practice only `Float64` works at present.
```

Comment:
A lot of our documentation has something like "in theory, everything is easy, but in practice, double precision leads makes life *interesting*".